doc/ChangeLog

```diff
@@ -1,4 +1,93 @@
 ===============================================================
+Tag name: ctsm5.1.dev173
+Originator(s): olyson (Keith Oleson,UCAR/TSS)
+Date: Thu 07 Mar 2024 12:07:43 PM MST
+One-line Summary: Improve vegetation health at high latitudes
+
+Purpose and description of changes
+----------------------------------
+
+The corresponding changes:
+
+Remove snicar_snobc_intmix from EXPERIMENTAL endrun (allow it to be true)
+Remove flg_snoage_scl in SNICAR such that xdrdt can have an effect (fixes #2298 )
+New parameter file and namelist values for clm5_1:
+
+- snow_thermal_cond_method = Sturm1997 (default for clm5_1)
+- snicar_snobc_intmix = .true. (default for clm5_1)
+- ctsm51_params.c240201.nc is the new CTSM parameter file (changes: froot_leaf(11:12)=1.2, FUN_fracfixers(11:12)=1, xdrdt=5, scvng_fct_mlt_sf=0.5, snw_rds_refrz=1500, fresh_snw_rds_max=400)
```

Comment:
ctsm51_params.c240208.nc is the version.  You probably had 240201 because I had that in my draft PR information, which I've since changed.

Reply:
Thank you for catching that!